torch/quantization/fake_quantize.py

```diff
@@ -13,23 +12,20 @@ class FakeQuantize(Module):
         `calcqparam`: A function that calculates quantization parameters
```

Comment:
the docstring is way out of date - please update

Reply:
Good catch, will update